google/auth/transport/__init__.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
(Sorry for just noticing, but the "All rights reserved." is no longer needed)

https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2390